src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.command.Command;
+import duke.common.exception.DukeException;
+import duke.data.Storage;
+import duke.task.TaskList;
+import duke.ui.Parser;
+import duke.ui.Ui;
```

Comment:
Good that you separate package import declaration, also import classes in detail.